Microsoft.Azure.CosmosRepository/src/Providers/DefaultCosmosContainerProvider.cs

```diff
@@ -1,96 +1,101 @@
-﻿// Copyright (c) IEvangelist. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Threading.Tasks;
-using Microsoft.Azure.Cosmos;
-using Microsoft.Azure.CosmosRepository.Options;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
+﻿// Copyright (c) IEvangelist. All rights reserved. Licensed under the MIT License.
 
 namespace Microsoft.Azure.CosmosRepository.Providers
 {
-    /// <inheritdoc/>
+    using System;
+    using System.Threading.Tasks;
+
+    using Microsoft.Azure.Cosmos;
+    using Microsoft.Azure.CosmosRepository.Options;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Options;
+
+    /// <inheritdoc />
     internal class DefaultCosmosContainerProvider<TItem>
         : ICosmosContainerProvider<TItem> where TItem : Item
     {
-        readonly Lazy<Task<Container>> _lazyContainer;
-        readonly RepositoryOptions _options;
-        readonly ICosmosClientProvider _cosmosClientProvider;
-        readonly ICosmosPartitionKeyPathProvider _cosmosPartitionKeyPathProvider;
-        readonly ILogger<DefaultCosmosContainerProvider<TItem>> _logger;
+        private readonly ICosmosClientProvider cosmosClientProvider;
```

Comment:
Again, all redundant access modifiers and `this` qualifiers should be reverted. Name private fields with `_` prefix, please.